diagnostic.md

```diff
@@ -73,14 +105,25 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --include --request POST http://www.example.com/donuts/FrenchCruller/3
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function(event){
+event.preventDefault();
+
+  let data = "French Cruller";
+  api.post(data)
```

Comment:
This won't work because `api` might not exist.  We want you to write a request using jQuery's `ajax()` method.

```javascript
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};
```